### PR TITLE
Add preliminary summit agenda

### PR DESCRIPTION
**Pull request by magnusbaeck**
### Applicable Issues
N/A

### Description of the Change
After a first meeting to set the summit agenda it's in sufficiently good shape to include in the summit page. See it live at https://magnusbaeck.github.io/eiffel-community.github.io/summit.html.

### Alternate Designs
None.

### Possible Drawbacks
None.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I    have the right to submit it under the open source license    indicated in the file; or

(b) The contribution is based upon previous work that, to the best    of my knowledge, is covered under an appropriate open source    license and I have the right under that license to submit that    work with modifications, whether created in whole or in part    by me, under the same open source license (unless I am    permitted to submit under a different license), as indicated    in the file; or

(c) The contribution was provided directly to me by some other    person who certified (a), (b) or (c) and I have not modified    it.

(d) I understand and agree that this project and the contribution    are public and that a record of the contribution (including all    personal information I submit with it, including my sign-off) is    maintained indefinitely and may be redistributed consistent with    this project or the open source license(s) involved.

Signed-off-by: Magnus Bäck \<magnus.back@axis.com>

### CODE REVIEW

**Comment by magnusbaeck** [comment]
cc @JiapengSun. I wasn't able to add you as a reviewer since you're not part of the eiffel-community organization (I've sent an invite).

File: summit.html [hunk 1]
```diff
@@ -48,12 +48,381 @@ <h1 class="section-heading">The practicalities</h1>
...
+                      authenticate their sender.
+                    </p>
+                    <p>
+                      <b>Speakers:</b> Magnus Bäck
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>15:00</td>
```
Comment (e-backmark-ericsson) [inline summit.html:164]: Maybe this should be 14.30? Currently two sessions start at 15. Are they in parallel?

**Review by e-backmark-ericsson — COMMENTED** [review]
Looks quite good to me, but please look over the times since they don't really match up. Otherwise approved from me.

File: summit.html (hunk 1 above)
Comment (JiapengSun) [reply summit.html:164]: Yes it shall be 1430

**Comment by JiapengSun** [comment]
Just a comment, we might need change lunch time to 1230 on both days due the availability of table, we can keep current agenda and update it later if that's the case.

File: summit.html (hunk 1 above)
Comment (magnusbaeck) [reply summit.html]: Yeah, several times were off. Update pushed.

**Comment by magnusbaeck** [comment]
> Just a comment, we might need change lunch time to 1230 on both days due the availability of table, we can keep current agenda and update it later if that's the case.

Yeah, shifting the lunch break shouldn't affect the schedule too much.

**Comment by JiapengSun** [comment]
Looks good! Approve from me :)

File: summit.html
Comment (m-linner-ericsson) [inline summit.html:1]: Shouldn't we have a link to the registration?

**Review by m-linner-ericsson — APPROVED** [review]
Looks ok from my side except that I wonder about adding a link to the registration form

**Review by JiapengSun — APPROVED** [review]

File: summit.html
Comment (magnusbaeck) [reply summit.html:1]: Doh, of course. Fixed. I'm not sure if the August 15 closing date for registrations has actually been decided, but we can adjust it later if necessary.

**Review by m-linner-ericsson — APPROVED** [review]

**Review by e-backmark-ericsson — APPROVED** [review]

File: summit.html
Comment (magnusbaeck) [reply summit.html:1]: Indeed, changed to August 22 in #67.

**Pull request merged**